app/views/case_workers/admin/management_information/_stats_report_with_date_form.html.haml

```diff
@@ -0,0 +1,22 @@
+- i18n_scope = 'case_workers.admin.management_information.index'
+
+- if report_rec&.started_at
+  = t('report_generated_at_html',
+      scope: i18n_scope,
+      datetime: report_rec.started_at.iso8601,
+      time: report_rec.started_at.strftime(Settings.report_date_format))
+
+  %p
+  = govuk_link_button(t('download', scope: i18n_scope),
+                      case_workers_admin_management_information_download_url(report_type: report_object.name, format: :csv))
+- else
+  = t('unavailable_report', scope: i18n_scope)
+  %p
```

Comment:
empty `<p>` tag

Reply:
should probably be a `<br≥`? There were spacing issues without this i believe

Reply:
have added a fix here 8016efdcf

Reply:
Remember we can use the spacing overrides: https://design-system.service.gov.uk/styles/spacing/